tests/console/yast2_clone_system.pm

```diff
@@ -38,7 +38,28 @@ sub run {
 
     # Check and upload profile for chained tests
     upload_asset "/root/autoinst.xml";
-    if (script_run 'xmllint --noout --relaxng /usr/share/YaST2/schema/autoyast/rng/profile.rng /root/autoinst.xml') {
+
+    unless (is_opensuse) {
+        my $sle15_repo;
+        if (is_sle('>15')) {
+            my $sp = (split(/-/, get_var('VERSION')))[-1];
```

Comment:
you doing same thing as @asdil12 in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/5577. maybe one of you should create function which could be used everywhere ?

Reply:
As the function would IMHO belong into `repo_tools.pm`, I will create this function.

Reply:
@asdil12 alright, ty!

Reply:
I updated https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/5577 to introduce a function `get_repo_var_name` (see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/5577#issuecomment-412835233).

It will work for `is_sle('>=15')` so you can also update your `if`/`else`.

Reply:
@mloviska I just merged #5577  so you can rebase and use a function